luma/core/interface/serial.py

```diff
@@ -295,3 +295,84 @@ def __setattr__(self, attr, val):  # pragma: no cover
 
     def __noop(self, *args, **kwargs):
         pass
+
+
+def ftdi_spi(device='ftdi://::/1', bus_speed_hz=12000000, CS=3, DC=5, RESET=6):
```

Comment:
i'd prefer `cs_pin`, `dc_pin` etc, or at least lowercase them.

Reply:
and can these methods not also be moved to `luma/core/interface/ftdi.py`? and import them here?

Reply:
Have used `gpio_CS`, `gpio_DC`, `gpio_RST` to align with same parameter names used by the SPI serial interface